# Xi Luo

- **Email**: luoxi2000@126.com  
- **GitHub**: [https://github.com/lxlovelxc](https://github.com/lxlovelxc)  
- **Address**: SUSTech, Shenzhen, Guangdong Province, China 518055

---

## Education

<table>
  <tr>
    <td>2022-2025</td>
    <td>Master of Science in Biology</td>
    <td>Southern University of Science and Technology</td>
    <td>GPA: 3.65/4.0</td>
    <td>Supervisor: Dr. Chen Xi</td>
  </tr>
  <tr>
    <td>2018-2022</td>
    <td>Bachelor of Science in Bioinformatics</td>
    <td>Southern University of Science and Technology</td>
    <td>GPA: 3.2/4.0</td>
    <td>Supervisor: Dr. Chen Xi</td>
  </tr>
  <tr>
    <td>Mar, 2024 -Jul, 2024</td>
    <td>Semester Exchange</td>
    <td>Peking University Shenzhen Graduate School</td>
    <td>GPA: 4.0/4.0</td>
    <td></td>
  </tr>
  <tr>
    <td>Jan,2020 -Jun, 2020</td>
    <td>Semester Exchange</td>
    <td>BISP, University of California, Berkeley</td>
    <td>Courses:  
      - Neurobiology of Disease (Helen Bateup)  
      - Quantitative Methods in Biology (Rasmus Nielsen)  
      - Biochemistry: Pathways, Mechanism, and Regulation (Michael Marletta)</td>
    <td></td>
  </tr>
</table>

---

## Research Experience

<table>
  <tr>
    <td>2022-2025</td>
    <td>Graduate Research</td>
    <td>Southern University of Science and Technology</td>
    <td>1/2.Collaborated with physician-scientist Dr.Wang Tianren at HKU Shenzhen Hospital on molecular differences between embryos from mothers of different ages. Performed scRNA-seq analysis on human embryos at post-implantation stage. Analyzed large-scale single-cell datasets of up to 6200 cells. Results under validation by Marta Shahbazi at MRC Laboratory of Molecular Biology, UK
    2/2.Single cell multiomics analysis including scRNA-seq and scATAC-seq.
    </td>
  </tr>
  <tr>
    <td>2021-2022</td>
    <td>Undergraduate Research</td>
    <td>Southern University of Science and Technology</td>
    <td>
      - Developed an interactive tool for single-cell transcriptome analysis using R Shiny  
      - Created a user-friendly web interface for non-bioinformatics users  
      - <a href="https://github.com/lxlovelxc">Project details available on GitHub</a>
    </td>
  </tr>
</table>

---

## Work Experience

<table>
  <tr>
    <td>2019-2020</td>
    <td>App Development Team Leader</td>
    <td>Freshman Project</td>
    <td>
      - Founded and led an interdisciplinary app development team for a biology-focused startup  
      - Hired and managed CS major and Business major students  
      - Gained valuable experience in entrepreneurship and team management
    </td>
  </tr>
</table>

---

## Conferences and Poster Presentations

<table>
  <tr>
    <td>Mar, 2024</td>
    <td>The Second International Conference on Single-cell and Spatial Omics</td>
  </tr>
  <tr>
    <td>Dec, 2023</td>
    <td>The Fourth Annual Meeting of EMBL Alumni in China</td>
  </tr>
  <tr>
    <td>Feb, 2023</td>
    <td>National Key Research on Biotechnology and Information Technology: Cell Atlas Project</td>
  </tr>
  <tr>
    <td>Nov, 2023</td>
    <td>Poster: Molecular Mechanisms Underlying Age-Related Changes in Early Post-Implantation Human Embryo Development (SUSTech)</td>
  </tr>
</table>

---

## Skills

### 1/3 Bioinformatics  
- **scRNA-seq** (STARsolo, Seurat, Linux):  
  Proficient in performing single-cell RNA sequencing (scRNA-seq) with experience in alignment, normalization, dimensionality reduction, and clustering. Skilled in using **STARsolo** and **Seurat** to identify cell subpopulations and differentially expressed genes. Expertise in **pseudobulk** methods for robust DE gene detection between conditions. Familiar with **pathway analysis** using **GSEA** and **clusterProfiler**.

- **scATAC-seq** analysis (Signac, Snakemake, Linux):  
  Experienced in analyzing scATAC-seq data using **Signac** and **Snakemake**. Able to perform alignment, peak calling, quality control, and downstream analysis in a Linux environment.


### 2/3 Programming  
- **R**:  
  Highly skilled in biostatistical analysis and handling high-dimensional biological datasets. Specialized in using **inferCNV** for copy number variation analysis and **miloDE** for differential expression in single-cell data.More code can be find on Github.

- **Shiny**:  
  Expert in building interactive web applications using **Shiny** for biological data visualization and exploration.


### 3/3 Data Visualization & Scientific Writing  
- **Adobe Illustrator**:  
  Skilled in producing high-quality scientific figures, posters, and visual content for conferences and papers. Proficient in creating schematic diagrams and other graphics to effectively communicate research findings.

- **LaTeX**:  
  Proficient in scientific writing and document formatting using **LaTeX**. Experienced in creating polished **Beamer presentations** and well-structured research papers to deliver precise and professional content in English.


### Additional Skills  
- **Soft Skills**:  
  Team leadership, Project management, Cross-disciplinary collaboration.


## Language Proficiency

- **TOEFL iBT Score**: 96  

---

## Career Goals

Aspiring to leverage my strong background in Biology, bioinformatics, and entrepreneurship to contribute to innovative research and development in the biotech industry. Seeking opportunities to apply my skills in data analysis, tool development, and cross-disciplinary collaboration to drive advancements in biological research and its practical applications.

---

## References

<table>
  <tr>
    <td>1. Dr. Chen Xi</td>
    <td>Professor of Systems Biology, Southern University of Science and Technology</td>
    <td>Email: chenx9@sustech.edu.cn</td>
  </tr>
  <tr>
    <td>2. Dr. Helen Bateup</td>
    <td>Professor of Neurobiology, University of California, Berkeley</td>
  </tr>
  <tr>
    <td>3. Dr. Rasmus Nielsen</td>
    <td>Professor of Integrative Biology, University of California, Berkeley</td>
  </tr>
  <tr>
    <td>4. Dr. Michael Marletta</td>
    <td>Professor of Chemistry and Molecular and Cell Biology, University of California, Berkeley</td>
  </tr>
</table>
